### ISSUE TITLE
Ghosting of backgrounds in Library grid view.

### ISSUE THREAD

**Issue by darkParable** [DirtDiglett/Pressure-for-Steam#357]
When re-sizing icons in the Library grid icon view, the background has ghosting of the different icon sizes. This also occurs when switching from the Store/Community/Profile pages back to the Library grid icon view, with afterimages of the previous page in the background of the grid view. I am running Pressure v1.0 on Windows 7 64-bit updated to the latest SP, and fonts are all installed. Steam is running on package version 1421729692, API v017.

![image](https://cloud.githubusercontent.com/assets/12549749/7756692/d69abb86-ffb1-11e4-8ea9-6d7e7a970a35.png)

![image](https://cloud.githubusercontent.com/assets/12549749/7756734/135cd446-ffb2-11e4-8559-15702b83f154.png)

**Comment by bykidi** [comment]
Make sure to update steam to latest BETA and download latest commit

**Comment by Connor37** [comment]
@darkParable I can't reproduce this.

**Comment by alvr** [comment]
It also happens to me.

![2015-05-22 14_42_36-steam](https://cloud.githubusercontent.com/assets/217315/7770625/da9a36fa-0090-11e5-89b1-e08fcfe094f2.png)

**Comment by bykidi** [comment]
@alvr what is your OS?

**Comment by alvr** [comment]
@bykidi The same as @darkParable Windows 7 64bits but Steam package version 1432242480 from May 20 and latest commit.

**Comment by darkParable** [comment]
I have the latest commit and also the latest Steam BETA update. Still having the same issue, only it extends to the other Library views as well, though this might have been going on since the original post. Closing and re-opening Steam window doesn't remove the artifacts, and only a full Steam restart wipes the artifacts away.

**Comment by DirtDiglett** [comment]
I've been a bit absent recently, but i'm back so i'll look into this and get it sorted out ASAP.

**Comment by DirtDiglett** [comment]
I'm pretty sure I know the cause:

Open the config.ini file and make sure that this section has one of the lines WITHOUT a // in front.

```
	//	GAME & OVERLAY BACKGROUNDS
	//	_______
	//	This section will let you disable the greenish background found on the Overlay, Game Details View and Game Grid View.
	//	Please choose one option below and delete the 2 forward slashes " // " at the start of that line.
	//	When you are done, make sure every OTHER line in this section is commented out with "//" at the start of each OTHER line.
			//include "includes/tweaks/backgrounds/enable.styles"
			include "includes/tweaks/backgrounds/disable.styles"
```

**Comment by darkParable** [comment]
@DirtDiglett I cleared out my Steam skin folder and re-downloaded the latest commit of Pressure, and the issues seems to have been resolved. I did take a look into the config.ini file after reinstallation, and indeed there is one line commented out of the section pasted above. Interestingly enough, I think my issue was caused by the config.ini file, or at least a lack of one. I didn't check before clean installing, but I don't think there was one in my old folder.

Anyhow, it seems to work now; I reinstalled a couple of days ago and have been using the skin with no issues so far. This issue could be closed now, I think, unless someone else is experiencing the same problem even with your advice..
